docs/source/user-guide/cli/installation.md

```diff
@@ -57,6 +57,8 @@ this to work.
 ```bash
```

Comment:
btw i was confused by the note about having to delete the .dockerignore file.
do you maybe know when this can be needed?

Reply:
I do not know what that documentation refers to

Reply:
i mean the line above

> Note that there is `.dockerignore` file in the root of the repository that may need to be deleted in order for this to work.

Reply:
Sorry - I meant that I didn't know to what it referred to